### DISCUSSION TITLE
Photok on SD-Card?

### DISCUSSION THREAD

**Discussion by IOSaw** [leonlatsch/Photok#230] [Q&A] [answered]
In the FAQ is written that _"Photok copies your photos into its local app storage"_ and my questions is about the possibility to bring that all to my SD-Card?

And additional (not concerning this subject....): Do you plan a similar App for videos too?

**Comment by leonlatsch** [comment, marked as answer]
Currently its not possible to change the storage location of Photok. But this feature was already requested and is planned -> #197 .

About your other question: Photok does indeed support videos. Just use the normal import and select "Video" in the Bottom Sheet Menu.

**Reply by IOSaw** [reply]
Thank you very much for your quick reply, Leon.

Do you have an time horizon for updating Photok with the requested feature? I agree with @hockeymikey from the linked thread, that it is a deal breaker on lack of internal memory.....

**Reply by leonlatsch** [reply]
Since Photok is in only maintained at this time, I cannot say when or if this will release. It will depend on if someone in the community implements it.

**Reply by IOSaw** [reply]
Thank you very much for your explanation, Leon. 🙏
